docs/configuration/filter-rules.md

```diff
@@ -141,7 +140,7 @@ allowing for more complex filtering.
 
 ## Dynamic Variables
 
-In addition to static values, you can also filter against _dynamic_ values using the following variables.
+In addition to static values, you can also filter against Dynamic Variables. The image below shows
```

Comment:
@erondpowell This a typo?

Reply:
Yes. a type error. Fixing it too along with the video commit.